tests/sentry/models/test_organizationmember.py

```diff
@@ -135,6 +136,38 @@ def test_delete_expired_clear(self):
         OrganizationMember.objects.delete_expired(timezone.now())
         assert OrganizationMember.objects.filter(id=member.id).first() is None
 
+    def test_delete_expired_SCIM_enabled(self):
+        organization = self.create_organization()
+        org2 = self.create_organization()
+        org3 = self.create_organization()
+        AuthProvider.objects.create(
+            provider="saml2", organization=organization, flags=AuthProvider.flags["scim_enabled"]
+        )
+        AuthProvider.objects.create(
+            provider="saml2", organization=org3, flags=AuthProvider.flags["allow_unlinked"]
+        )
+        AuthProvider.objects.create(
```

Comment:
@maxiuyuan This `AuthProvider` and `org2` seems unused for this test. We could probably delete them.